src/widgets/index/__tests__/index-test.ts

```diff
@@ -1367,15 +1369,73 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/index/js/"
           .setQueryParameter('page', 5);
 
         expect(instance.getWidgetState({})).toEqual({
-          indexName: {
+          indexId: {
             query: 'Apple',
             page: 5,
           },
         });
       });
 
-      it('uses `indexId` for scope key', () => {
-        const instance = index({ indexName: 'indexName', indexId: 'indexId' });
+      it('uses `uiState` for the local `uiState`', () => {
```

Comment:
Can we make this test name clearer? Do you mean "forwards the init `uiState` to the widget state"?

Reply:
`uses the provided `uiState` for the local `uiState` is it better?

Reply:
Yep!

Reply:
https://github.com/algolia/instantsearch.js/pull/4074/commits/2cf0ac103c47dcc864c14a6a3b87ca1ec0e31660